cmd/copy.go

```diff
@@ -298,10 +298,6 @@ func (raw rawCopyCmdArgs) cook() (cookedCopyCmdArgs, error) {
 	// Example1: for Local to Blob, preserve-last-modified-time flag should not be set to true
 	// Example2: for Blob to Local, follow-symlinks, blob-tier flags should not be provided with values.
 	switch cooked.fromTo {
-	case common.EFromTo.LocalBlobFS():
```

Comment:
There were 6 things in this check that's been removed, and only 5 of them relate to this PR. Shouldn't the check remain just for the other one (namely BlobType)?

Reply:
This point appears to be still unresolved?

Reply:
Only one should remain, you're right.